<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QMainWindow" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>275</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>275</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>275</width>
     <height>430</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>275</width>
     <height>430</height>
    </size>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>281</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_menu">
     <attribute name="title">
      <string>Menu</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_biogas_calculator">
         <property name="text">
          <string>Biogas Calculator</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>7</pointsize>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Simulate production for specific plant</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_personal_data">
     <attribute name="title">
      <string>Personal Data</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>User:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_username">
         <property name="text">
          <string>User</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_personal_data">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>271</width>
        <height>201</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string>Change Personal Data</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>261</width>
         <height>131</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_name">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maxLength">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Surname</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_surname">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maxLength">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>E-Mail</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_email">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maxLength">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton_update_data">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>160</y>
         <width>151</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Change Data</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_personal_data_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>250</y>
        <width>271</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string>Change Password</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>261</width>
         <height>88</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Old Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_old_password">
            <property name="maximumSize">
             <size>
              <width>160</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maxLength">
             <number>30</number>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>New Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_new_password">
            <property name="maximumSize">
             <size>
              <width>160</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maxLength">
             <number>30</number>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::PasswordEchoOnEdit</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton_change_password">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>120</y>
         <width>151</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Change Password</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_contact_data">
     <attribute name="title">
      <string>Contact Data</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_personal_data_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>271</width>
        <height>251</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string>Corespondance Adress</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>261</width>
         <height>171</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Street</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_street">
            <property name="maximumSize">
             <size>
              <width>180</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maxLength">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Number</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_adress_number">
            <property name="maximumSize">
             <size>
              <width>180</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maxLength">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>City</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_city">
            <property name="maximumSize">
             <size>
              <width>180</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maxLength">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Postal Code</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_postal_code">
            <property name="maximumSize">
             <size>
              <width>180</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maxLength">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Country</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_country">
            <property name="maximumSize">
             <size>
              <width>180</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maxLength">
             <number>50</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton_edit_address">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>210</y>
         <width>151</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Change/Add Address</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>280</y>
        <width>271</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>Phone Numbers</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QPushButton" name="pushButton_phone_numbers">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>View Phone Numbers</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>275</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>5</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
